Sparkle/SUStatusController.m

```diff
@@ -51,6 +52,11 @@ - (void)windowDidLoad
     [[self window] center];
     [[self window] setFrameAutosaveName:@"SUStatusFrame"];
     [self.progressBar setUsesThreadedAnimation:YES];
+
+    if ([NSFont respondsToSelector:@selector(monospacedDigitSystemFontOfSize:weight:)])
```

Comment:
Use `+[SUOperatingSystem isOperatingSystemAtLeastVersion:]` instead and suppress the `-Wpartial-availability` warnings (like in `-[SUUIBasedUpdateDriver localizedStringFromByteCount:]`). Apple doesn't in general recommend doing runtime checking anymore (with some exceptions..) because private and incomplete implementations can exist in prior OS versions.